pkg/cloudprovider/registry/aws.go

```diff
@@ -1,3 +1,4 @@
+//go:build aws
```

Comment:
What's going on here?

Reply:
Looks like it's to do with Go 1.17.2 - it's getting added by `make dev`